app/components/classCard.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { View, Text, Image, StyleSheet, TouchableOpacity } from 'react-native';
+import Colors from '../utils/Colors';
+
+interface ClassCardProps {
+  time: string;
+  title: string;
+  members: string;
+  year: string;
+  status: string;
+  imageUrl: string;
+  onPress: () => void;
+}
+
```

Comment:
Please change the indentation from 2 to 4 spaces. VSCode can easily do that for you.

Reply:
done I think